### ISSUE TITLE
[v3.0.3-rc.7] iOS badge not updating

### ISSUE THREAD

**Issue by v3rron** [raix/push#260]
Since the upgrade to `v3.0.3-rc.7` and migrating from Google GCM to Firebase FCM, iOS notification badge is stuck to one number. When clicking on app icon with a badge number of 3, it doesn't disappear afterwards. Sometimes it will change to a smaller or higher number, but never disappears completely.

**Comment by aqib1604** [comment]
@v3rron I have a same issue. :(

**Comment by yuwanlin** [comment]
Yes, It's. `v3.1.0-pre.1`. when IOS client receive a notification(without open the app),the badge is always `1`.But It will be changed if I `Push.setBadge(count)`,It's a absolute value. May I listen the notification event ??? @raix
